### PR TITLE
Update enabled since v1.25

### PR DESCRIPTION
**Pull request by stephanos**
## What does this PR do?

Now that [Server 1.25 has been release](https://github.com/temporalio/temporal/releases/tag/v1.25.0) with Workflow Update enabled by default, the installation instructions need a qualifier.

### CODE REVIEW

**Comment by fairlydurable** [comment]
Kicking this over to @dandavison

File: docs/encyclopedia/application-message-passing.mdx [hunk 1]
```diff
@@ -141,7 +141,7 @@ Signal-With-Start is a great tool for lazily initializing Workflows. When you se
 
 :::note
 
-Workflow Updates are currently disabled by default on Temporal Server.
+Before v1.25.0, Workflow Updates were disabled by default on Temporal Server.
```
Comment (dandavison) [inline docs/encyclopedia/application-message-passing.mdx:144]: If this statement is to remain for all time, would it be more normal to say something like

> Workflow Update was enabled by default in v1.25.0.

? (Sorry to debate a one sentence PR!)

Comment (stephanos) [reply docs/encyclopedia/application-message-passing.mdx:144]: 👍 agree, that's better!

Comment (stephanos) [reply docs/encyclopedia/application-message-passing.mdx]: Looking at the context again, I think the purpose of this note is to alert people that before 1.25, Update needs to be manually enabled. I reworded it to make that clearer.

**Review by dandavison — APPROVED** [review]

File: docs/encyclopedia/application-message-passing.mdx (hunk 1 above)
Comment (dandavison) [reply docs/encyclopedia/application-message-passing.mdx]: LGTM

**Comment by stephanos** [comment]
@fairlydurable sorry, I haven't made any documentation changes in almost a year. Will you get this merged or am I? I'm a bit confused by the Vercel step failure.

**Comment by fairlydurable** [comment]
Got your notice, saw that everything seemed clarified. Please always resolve conversations so we know there's nothing left.

**Review by fairlydurable — APPROVED** [review]
Unblocking this for merge.

**Pull request merged**